linera-execution/src/execution.rs

```diff
@@ -232,10 +241,16 @@ where
         };
         // Set the authenticated signer to be used in outgoing messages.
         result.authenticated_signer = signer;
-        *remaining_fuel = runtime.remaining_fuel();
+        *runtime_meter = runtime.runtime_meter();
+        if runtime_meter.bytes_read > runtime_meter.maximum_bytes_read {
+            return Err(ExecutionError::ExcessiveRead);
+        }
+        if runtime_meter.bytes_write > runtime_meter.maximum_bytes_write {
+            return Err(ExecutionError::ExcessiveWrite);
+        }
```

Comment:
We'll need to make sure that the maximum can't be exceeded (at least not by too much) within each call to `execute_operation` or `execute_message`, not just afterwards. (Could be a separate PR, of course.)

Reply:
This has been addressed in subsequent work of this PR.